From 8d0df32f3410e5b0bd55fe974e3b27878703d8d7 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 7 Dec 2018 16:21:13 +0900
Subject: [PATCH 622/909] arm64: dts: r8a77990-es10-ebisu-4d: Add lossy phandle
 to MMNGR driver node

commit 6a682d5b4e164ebf17b218db13f12f00fa05f1e1 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds the lossy descompress area phandle to MMNGR
driver node for R8A77990 ES1.0 SoC on ebisu board, and allow it to parse
reserved memory for lossy compression feature.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu-4d.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu-4d.dts b/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu-4d.dts
index e831a0ef6dcd..dff31f1ec3ad 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu-4d.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-es10-ebisu-4d.dts
@@ -25,6 +25,10 @@
 			reg = <0x00000000 0x54000000 0x0 0x03000000>;
 		};
 	};
+
+	mmngr {
+		memory-region = <&mmp_reserved>, <&lossy_decompress>;
+	};
 };
 
 &pciec0 {
-- 
2.17.1

